cypress/composables/workflow.ts

```diff
@@ -358,3 +362,7 @@ export function openContextMenu(
 export function clickContextMenuAction(action: string) {
 	getContextMenuAction(action).click({ force: true });
 }
+
+export function clickClearExecutionDataButton() {
```

Comment:
Would it make sense to combine both of these into a "clearExecution" composable here? If we don't need to get/click individually just making it a single action would be nicer.